server/controllers/todoController.js

```diff
@@ -1,40 +1,55 @@
-const { body, validationResult } = require("express-validator");
-const db = require("../db/connect.js");
+const { body, validationResult } = require('express-validator');
+const db = require('../db/connect.js');
 
+// Client access: NOTE - if you want to interact with the database,
+// you first need to get a reference to the collection (think of a table in SQL).
+// To do this, you have to await the return of calling client.db().collection('todos') and then you can do what you need off the
+// collection - see the example below in the POST request handler.
 const client = db.getClient();
 
 // post todo's to db
 exports.todo_post = [
   // validate & sanatize data
-  body("item", "You you need to add an item")
+  body('item', 'You you need to add an item')
```

Comment:
Try to enforce a standard for coding in your team that everyone agrees on. (single quotes vs double quotes) semicolons or no semi ETC. this prevents such cases like these